www/css/main.diary.css

```diff
@@ -113,20 +113,26 @@ a.item-content {
 .btn-input-confirm-red,
 .btn-input-confirm-red:hover,
 .btn-input-confirm-red:active {
-  background-color: #ED2D3A;
-  color: white;
+  background-color: white;
+  color: #0088ce;
+  border-color: #0088ce;
+  box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
 }
 .btn-input-confirm-yellow,
 .btn-input-confirm-yellow:hover,
 .btn-input-confirm-yellow:active {
-  background-color: #FFC108;
+  background-color: rgba(255, 165, 0, 0.8);  /* tentatively orange for now */
```

Comment:
I have changed this to a shade of orange/yellow because I had found from family and friend feedback that this yellow looked more appealing than the yellow that was in the original color scheme. I simply found what the RGBA color value was for orange, and found the a opacity that looked appealing.

Reply:
This is the color used for the yellow guessed-labels

Before it was this shade #ffc108 (official NREL yellow)
<img width="201" alt="Screen Shot 2022-08-31 at 11 10 17 AM" src="https://user-images.githubusercontent.com/61334340/187742895-99395fbc-945b-4207-83e7-be5d0dbe2224.png">


Now it is this shade $ffa500 (custom picked orange-yellow)
<img width="169" alt="Screen Shot 2022-08-31 at 11 10 56 AM" src="https://user-images.githubusercontent.com/61334340/187742882-1beefd3f-ecb8-4153-b847-3bd77bfe4618.png">

There is also an NREL official orange #fd7e14
<img width="190" alt="Screen Shot 2022-08-31 at 11 39 09 AM" src="https://user-images.githubusercontent.com/61334340/187744404-082c1b1d-800e-4bf6-a9cd-75167650cc70.png">

They are all very similar, I don't think it would make a huge difference what the color is between these 3. That being said, if it is important to use only NREL official shades, we should opt for one of those instead. @shankari what is your opinion on these 3 shades?

Reply:
as we saw in the meeting with NREL comms, a lot of thought goes into accessibility and color choice. I think that is better than an informal poll of tyler's friends and family. Please use one of the official colors. For the record, my daughter likes the official yellow, but I don't care.